src/data/complex/exponential.lean

```diff
@@ -1351,6 +1352,18 @@ begin
   convert exp_bound hxc hn; norm_cast
 end
 
+lemma exp_bound' {x : ℝ} (h1 : 0 ≤ x) (h2 : x ≤ 1) {n : ℕ} (hn : 0 < n) :
+  real.exp x ≤ ∑ (m : ℕ) in finset.range n, x ^ m / (m.factorial)
+  + x ^ n * ((n : ℝ) + 1) / (n.factorial * (n : ℝ) ) :=
+begin
+  have h3 : |x| = x, simp, exact h1,
```

Comment:
```suggestion
  have h3 : |x| = x := by simpa,
```

This line is against mathlib style in a couple ways:

1. We very rarely put more than one comma'd expression in a line.
2. (far more important) a `simp` that doesn't close the goal (so called "non-terminal `simp`") shouldn't exist, as `simp` lemmas can change in the future, breaking a proof. There is many ways to fight against this (including the excellent `squeeze_simp`, which outputs an equivalent `simp only` expression, but here you can replacte this `simp` with `simpa`, which is `simp + assumption` (actually a bit cleverer, but meh), and then it does close a goal so it's fine